src/demo_05_post_pending_transfers.zig

```diff
@@ -6,27 +6,27 @@ const Transfer = tb.Transfer;
 pub fn main() !void {
     const commits = [_]Transfer{
         Transfer{
-            .id = 1004,
-            .pending_id = 1001,
-            .ledger = 1,
+            .id = 2001,
             .debit_account_id = 1,
             .credit_account_id = 2,
             .user_data = 0,
             .reserved = 0,
+            .pending_id = 1001,
```

Comment:
Nice fix on the field order here, to keep it consistent with the actual struct order.

Reply:
:bow: